src/platforms/macos/macosmenubar.cpp

```diff
@@ -63,7 +65,10 @@ void MacOSMenuBar::initialize() {
   m_preferencesAction->setMenuRole(QAction::PreferencesRole);
   m_preferencesAction->setVisible(vpn->state() == MozillaVPN::StateMain);
 
-  m_closeAction = fileMenu->addAction("", vpn->controller(), &Controller::quit);
```

Comment:
Should we keep this too or not?

Reply:
We have a separate quit action on line 50 (52 after the patch) which does the same thing.
So we do not need to keep it.